ranked-choice-voting/index.py

```diff
@@ -17,6 +18,20 @@ def create_dataset_for_modeling(num_ballots_to_generate):
                          sort=False)
 
     master_df = master_df.drop_duplicates()
+
+    # Drop all cols with "Rounds" info:
+    to_filter_out = [col for col in master_df if not col.startswith('Round')]
+    master_df  = master_df[to_filter_out]
+
+    # Drop other cols we don't :
+    master_df = master_df.drop(columns=['Election', 'filename',
+                                        'pyrankvote_winner_x',
+                                        'condorcet_winner',
+                                        'pyrankvote_winner_y'])
+
+    # Vectorizing dependent var:
+    master_df['spoiled'] = np.where(master_df['spoiled'] == 'Y', 0, 1)  # 0 = yes, 1 = no
```

Comment:
Should it be the other way round? Spoiled? Yes:1 , No:0?

Reply:
yeah, I agree with you Sherin, but we can change later/at any point.